src/Data/Models/Wallet.cs

```diff
@@ -86,9 +102,9 @@ public class Wallet : Entity
         
 
         /// <summary>
-        /// The internal wallet is used to co-sign with other keys of the wallet entity
+        /// The internal wallet is used to co-sign with other keys of the wallet entity. It is optional for imported BIP39 wallets
         /// </summary>
-        public int InternalWalletId { get; set; }
+        public int? InternalWalletId { get; set; }
 
         public InternalWallet InternalWallet { get; set; }
```

Comment:
Shouldnt this be optional aswell?

Reply:
? Was added to inform the DB model for the FK to be nullable, but true

Reply:
```suggestion
        public InternalWallet? InternalWallet { get; set; }
```